### PR TITLE
Combine metrics to a single doc

### PR DESCRIPTION
**Pull request by yyyogev**

### CODE REVIEW

File: src/main/java/io/logz/jmx2logzio/clients/JolokiaClient.java [hunk 1]
```diff
@@ -143,20 +146,20 @@ public void setExtraDimensions(List<Dimension> extraDimensions) {
      * @param response A response map from the Jolokia server
      * @return a list of logz.io metrics
      */
-    private List<Metric> getMetricsForResponse(Map<String, Object> response) {
+    private Metric getMetricsForResponse(Map<String, Object> response) {
```
Comment (idohalevi) [inline src/main/java/io/logz/jmx2logzio/clients/JolokiaClient.java]:
```suggestion
    private Metric getMetricForResponse(Map<String, Object> response) {
```

File: src/main/java/io/logz/jmx2logzio/objects/Metric.java [hunk 2]
```diff
@@ -6,10 +6,10 @@
 import java.time.Instant;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 public class Metric {
```
Comment (idohalevi) [inline src/main/java/io/logz/jmx2logzio/objects/Metric.java]: @yyyogev Do you think the number of parameters can grow in the future?

File: src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java [hunk 3]
```diff
@@ -68,7 +70,9 @@ public void sendMetricTest() {
         List<Dimension> dimensions = new ArrayList<>();
         dimensions.add(0, new Dimension("type", "myType"));
         List<Metric> metrics = new ArrayList<>();
-        metrics.add(new Metric(key, value, Instant.now(), dimensions));
+        Map<String, Number> keyvalues =  new HashMap<>();
+        keyvalues.put(key,value);
```
Comment (idohalevi) [inline src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java]: space

File: src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java [hunk 4]
```diff
@@ -68,7 +70,9 @@ public void sendMetricTest() {
         List<Dimension> dimensions = new ArrayList<>();
         dimensions.add(0, new Dimension("type", "myType"));
         List<Metric> metrics = new ArrayList<>();
-        metrics.add(new Metric(key, value, Instant.now(), dimensions));
+        Map<String, Number> keyvalues =  new HashMap<>();
```
Comment (idohalevi) [inline src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java]: What does `keyvalues` represent?

File: src/main/java/io/logz/jmx2logzio/clients/JolokiaClient.java (hunk 1 above)
Comment (yyyogev) [reply src/main/java/io/logz/jmx2logzio/clients/JolokiaClient.java]: I'm not sure if we should change the name.. It is a few metrics in a single doc..

File: src/main/java/io/logz/jmx2logzio/objects/Metric.java (hunk 2 above)
Comment (yyyogev) [reply src/main/java/io/logz/jmx2logzio/objects/Metric.java]: Not sure I follow

File: src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java (hunk 4 above)
Comment (yyyogev) [reply src/test/java/io/logz/jmx2logzio/LogzioSenderTest.java]: maps to metric names to values

**Pull request merged**